elide-annotations/src/main/java/com/yahoo/elide/annotation/Include.java

```diff
@@ -32,4 +32,10 @@
      * @return the string
      */
     String type() default "";
+
+    /**
+     * (Optional) Whether or not this class inherits permissions from a super class.
+     * @return
+     */
+    boolean inheritPermissions() default true;
```

Comment:
There is another norm that might be the common case though.  That is having mapped super classes and sticking permissions in them that users want to be inherited.  I'm not sure how common our microservice plan is.  It seems like something that a mature organization might encounter - but most folks I would expect to have a single service.  Anyway, in the absence of knowing what is truly comment, I would vote to not break the contract.